Add AI Covenant (#1)
chartgerink/ai-covenant: content/_index.md

---
title: AI Covenant
toc: false
---

![100% human made](https://img.shields.io/badge/100%25%20human%20made-violet?style=for-the-badge&logo=myspace)

The use of "Artificial Intelligence" models affects the dynamics between people. We encourage mindful and considerate use of "AI" in our community that is grounded in our community values.

Specific behaviors we do not allow:

> - Using AI tools without the consent of people affected by them.

Reviewer: ```suggestion

```

Reviewer: (We added this one at the same time )

- Using AI tools without the consent of the people affected
- Passing AI generated content off as your own
- Arriving to a meeting with an AI assistant without asking the other participants
- Sending your AI assistant to meetings in your place
- If the use of generative tools has been allowed, not verifying or validating the generated content.

Specific behaviors we encourage/allow:

> - Marking any generated content with the model and prompt used for its creation (for example images).

Reviewer: ```suggestion
- Mark any generated content with the model and prompt used for its creation (for example images).
```

Reviewer: Just for verb conjugation consistency

> - Requesting consent of participants and organizers to use AI based tools.

Reviewer: ```suggestion
- Request consent of participants and organizers to use AI based tools.
```

---
title: AI Covenant
toc: false
---

The use of "Artificial Intelligence" models affects the dynamics between people. We encourage mindful and considerate use of "AI" in our community that is grounded in our community values.

Specific behaviors we do not allow:

> 

Reviewer: ```suggestion
- Using AI tools without the consent of people affected by them.
```

> - Passing generated content off as your own

Reviewer: ```suggestion
- Passing AI generated content off as your own
```

> - Sending your AI assistant to meetings in your place

Reviewer: ```suggestion
- Arriving to a meeting with an AI assistant without asking the other participants
- Sending your AI assistant to meetings in your place
```

> - [promoting the use of AI for everything (AI-ifying everything)]

Reviewer: This point is kind of covered below when we said "Justify the use of "AI" on a case by case basis, taking into account risks, harms."

> - [not verifying or validating generated content]

Reviewer: ```suggestion
- - If the use of generative tools has been allowed, not verifying or validating the generated content.
```

- [fully AI based production cycles]

Specific behaviors we encourage/allow:

> 

Reviewer: ```suggestion
- If the use of AI tools is allowed, be explicit when/how they have been used (e.g. for meeting transcription, for content summarization, etc.)
```

Author: Hm I figure we use a lot of AI unknowingly everyday - would we also have to disclose this? I don't think grammar checks for example are considered under the same header by many people.

What I am trying to get at: How deep/prescriptive would we want to get with listing all (AI) tools used? I don't really know - because using it to ideate is not that problematic but using the output one to one is.

Reviewer: Agree!

Reviewer: (I removed that example)

> - Generated content is specifically marked with the model + prompt (for example images)

Reviewer: ```suggestion
- Marking any generated content with the model and prompt used for its creation (for example images).
```

> - Requesting consent to...

Reviewer: ```suggestion
- Requesting consent of participants and organizers to use AI based tools.
```

  - ...use "AI"
  - ...share [information] with "AI"
- Justify the use of "AI" on a case by case basis, taking into account risks, harms.

> 

Reviewer: ```suggestion
- If an AI tool has been used with the consent of the group for transcription purposes, allow participants in that conversation to review the result before sharing it with the intended audience.
```

> 

Author: ```suggestion
- If an AI tool is used (with consent) to generate information about someone (e.g., transcription), this person is permitted to correct the information.
```

---
title: AI Covenant
toc: false
---

The use of "Artificial Intelligence" models affects the dynamics between people. We encourage mindful and considerate use of "AI" in our community that is grounded in our community values.

Specific behaviors we do not allow:

- Passing generated content off as your own
- Arriving to a meeting with an AI assistant without asking the other participants
- Sending your AI assistant to meetings in your place

> - [promoting the use of AI for everything (AI-ifying everything)]

Author: ```suggestion
```

- - If the use of generative tools has been allowed, not verifying or validating the generated content.
- [fully AI based production cycles]

Specific behaviors we encourage/allow:

- Generated content is specifically marked with the model + prompt (for example images)
- Requesting consent of participants and organizers to use AI based tools.
  - ...use "AI"

>   - ...share [information] with "AI"

Reviewer: Inform other participants if data or content they have produced is going to be shared with an "AI" tool (e.g., for content summarization)

>   - ...share [information] with "AI"

Reviewer: ```suggestion
- Inform other participants if data or content they have produced is going to be shared with an "AI" tool (e.g., for content summarization)
```
